plugins/colorschemes/bamboo.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  ...
+}:
+let
+  inherit (lib.nixvim) toLuaObject;
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "bamboo";
+  isColorscheme = true;
+  packPathName = "bamboo.nvim";
+  package = "bamboo-nvim";
+  # The colorscheme option is set by the `setup` function.
+  colorscheme = null;
+  callSetup = false;
```

Comment:
Might make more sense to have these just above `extraConfig`, since that is where you are manually implementing this stuff

Reply:
Moved as suggested just above the extraConfig section.